docs/sdk_developers/training/coding_token_transactions.md

```
@@ -65,6 +65,24 @@ def set_account_id(self, account_id: AccountId) -> "TokenAssociateTransaction":
+This feature enables chaining.
+
+For example:
+
+### Standard Usage
+
+```python
+tx.set_account_id(account_id)
+tx.set_token_id(token_id)
+tx.freeze()
+tx.execute(client)
+```
```

markdownlint-cli2 (0.18.1)

Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)